1.10/installing/custom/configuration-parameters.md

```diff
@@ -63,6 +63,27 @@ specified.
   This parameter specifies where to permanently store DC/OS packages. The value must be a file URL,
   for example, `file:///var/lib/dcos/cosmos/packages`.
 
+### dns_bind_ip_blacklist
+This parameter is a list of IP addresses that DC/OS DNS resolvers will not bind to.
+
+### dns_forward_zones
+This parameter is used by DC/OS DNS resolvers to configure the forwarding of queries for DNS addresses in a particular DNS zone to a custom DNS resolver.
```

Comment:
How about: `This parameter specifies a YAML nested list (`-`) of DNS addresses and zones whose query requests are forwarded to a custom resolver.`